sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -83,6 +92,7 @@
 
     private final ClientLogger logger = new ClientLogger(DataLakeFileSystemAsyncClient.class);
     private final DataLakeStorageClientImpl azureDataLakeStorage;
+    private final DataLakeStorageClientImpl blobDataLakeStorageFs;
```

Comment:
Can we add a comment here that this is just for listBlobs/Paths

Reply:
Done